controllers/challenges.js

```diff
@@ -5,11 +5,33 @@ const Challenge = require('../models').models.Challenge;
 const axios = require("../lib/axios");
 
 exports.getChallenges = function* (next) {
-
+  this.type = 'json';
+  try {
+    const challenges = yield Challenge.find();
+      this.status = 200;
+      this.body = {
```

Comment:
Does this respect the apiary documentation? As I read, this should return an array of challenges `[]` not `{challenges:[]}`